OmgvaPOS/src/HelperUtils/JwtTokenHandler.cs

```diff
@@ -47,42 +55,63 @@ public static bool CanDeleteUser(string tokenString, long businessId, long userI
             return true;
         }
 
-        public static long? GetTokenBusinessId(string tokenString) {
+        public static long? GetTokenBusinessId(string tokenString)
+        {
             var token = GetJwtToken(tokenString);
-            try {
+            try
+            {
                 return long.Parse(token.Claims.FirstOrDefault(c => c.Type.Equals("http://schemas.xmlsoap.org/ws/2005/05/identity/claims/sid"))!.Value);
             }
-            catch (Exception) {
+            catch (Exception)
+            {
                 return null;
             }
         }
 
-        public static long? GetTokenUserId(string tokenString) {
+        public static long? GetTokenUserId(string tokenString)
+        {
             var token = GetJwtToken(tokenString);
-            try {
+            try
+            {
                 return long.Parse(token.Claims.FirstOrDefault(c => c.Type.Equals("http://schemas.xmlsoap.org/ws/2005/05/identity/claims/nameidentifier"))!.Value);
             }
-            catch (Exception) {
+            catch (Exception)
+            {
                 return null;
             }
         }
+        public static TokenDetailsDTO GetTokenDetails(string tokenString)
+        {
+            var token = GetJwtToken(tokenString);
+            TokenDetailsDTO tokenDetails = new()
+            {
+                Name = token.Claims.FirstOrDefault(c => c.Type.Equals("http://schemas.xmlsoap.org/ws/2005/05/identity/claims/name")).Value,
+                NameIdentifier = token.Claims.FirstOrDefault(c => c.Type.Equals("http://schemas.xmlsoap.org/ws/2005/05/identity/claims/nameidentifier")).Value,
+                Role = token.Claims.FirstOrDefault(c => c.Type.Equals("http://schemas.microsoft.com/ws/2008/06/identity/claims/role")).Value
+            };
```

Comment:
Please use the same constants like `ClaimTypes.NameIdentifier` that are used when creating the JWT instead of hardcoding the string